modules/programs/zathura.nix

```diff
@@ -0,0 +1,53 @@
+{ config, lib, pkgs, ...}:
+
+with lib;
+
+let
+
+  cfg = config.programs.zathura;
+
+  formatLine = n: v:
+    let
+      formatValue = v:
+        if isBool v then (if v then "true" else "false")
+        else toString v;
+    in
+      "set ${n}\t\"${formatValue v}\"";
+in
+
+{
+  meta.maintainers = [maintainers.rprospero];
+
+  options.programs.zathura = {
+    enable = mkEnableOption ''Zathura is a highly customizable and funtional
```

Comment:
The mkEnableOption will produce a text of the form "Whether to enable ${arg}." where `arg` is the text you provide. So this will become "Whether to enable Zathura is a highly customizable and funtional document viewer focused on keyboard interaction." which isn't very grammatically correct 🙂

I'd suggest
```nix
mkEnableOption ''
  Zathura, a highly customizable and funtional document
  viewer focused on keyboard interaction''
```

Unfortunately `mkEnableOption` doesn't strip whitespace from the end of the text so cannot put `''` on it's own line, otherwise there will be an unfortunate space: `… interaction .`

Reply:
I've changed the enable text to your suggestion.